WordPress/src/wordpress/java/org/wordpress/android/ui/accounts/login/LoginPrologueRevampedFragment.kt

```diff
@@ -0,0 +1,98 @@
+package org.wordpress.android.ui.accounts.login
+
+import android.content.Context
+import android.content.res.Configuration.UI_MODE_NIGHT_YES
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.compose.foundation.Image
+import androidx.compose.foundation.background
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.padding
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.layout.ContentScale.Companion
+import androidx.compose.ui.platform.ComposeView
+import androidx.compose.ui.res.colorResource
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import androidx.fragment.app.Fragment
+import org.wordpress.android.R.color
+import org.wordpress.android.R.drawable
+import org.wordpress.android.R.string
+import org.wordpress.android.ui.accounts.login.compose.components.Tagline
+import org.wordpress.android.ui.accounts.login.compose.components.PrimaryButton
+import org.wordpress.android.ui.accounts.login.compose.components.SecondaryButton
+import org.wordpress.android.ui.compose.theme.AppTheme
+
+class LoginPrologueRevampedFragment: Fragment() {
+    private lateinit var loginPrologueListener: LoginPrologueListener
+
+    override fun onCreateView(
+        inflater: LayoutInflater,
+        container: ViewGroup?,
+        savedInstanceState: Bundle?
+    ): View = ComposeView(requireContext()).apply {
+        setContent {
+            AppTheme {
+                LoginScreenRevamped(
+                        onLoginClicked = { loginPrologueListener.showEmailLoginScreen() }
+                )
+            }
+        }
+    }
+    companion object {
+        const val TAG = "login_prologue_revamped_fragment_tag"
+    }
+
+    @Suppress("TooGenericExceptionThrown")
```

Comment:
I believe we can throw a different exception to get rid of this suppression.
I've been trying to throw an `IllegalStateException` in my WIP work on the JP landing screen revamp and then detekt suggested to use `error("message_here")`:

<img width="790" alt="Screenshot 2022-08-31 at 17 00 51" src="https://user-images.githubusercontent.com/4588074/187711497-3039f7fa-2024-4e82-bf06-76b725358ca9.png">

Now that I've played with `check` I tend believe it's even nicer than [`error()` inside an `if` statement](https://github.com/wordpress-mobile/WordPress-Android/compare/trunk...issue/17073-jetpack-landing-screen-revamp#diff-0ddbcac04d7f8e41dd4ecdc185e45a7b357addd9d9822210e14ab73c2267bccdR51-R53)

<img width="905" alt="Screenshot 2022-08-31 at 17 03 19" src="https://user-images.githubusercontent.com/4588074/187712131-997ffe99-b5dc-4d03-abb7-e4ba9c0b64ff.png">

Reply:
That sounds great Ovi! I hadn't given this much thought, so I'm glad you brought it up.

Reply:
Addressed here: https://github.com/wordpress-mobile/WordPress-Android/pull/17104/commits/1ebcc6c17664ac5455a1ea44b81c7899c48cafc6